### DISCUSSION TITLE
Puzzle 13 error in expected output

### DISCUSSION THREAD

**Discussion by ccammers** [smpetersgithub/AdvancedSQLPuzzles#35] [General]
the last value on the output  7/5/18 would be -25 or adjust the input for 7/5 to be -50. The input adjustment quantities and the output don't match.

Thanks!!

PS I really enjoyed the puzzles so far. I've been solving them in Qlik Load script as an exercise to mentor some junior developers. the one on permutations was a big challenge.

**Comment by smpetersgithub** [comment]
Thank you!  I will fix the puzzle.  Let me know if you find any other errors.

**Comment by smpetersgithub** [comment]
Closing
